src/core/ipfs/files.js

```diff
@@ -81,6 +82,10 @@ module.exports = function files (self) {
           return file
```

Comment:
yeah, data can be lost if I don't pause it, oh my love for node streams is so great